### DISCUSSION TITLE
Please share proguard rules to apply for apktool.jar

### DISCUSSION THREAD

**Discussion by jewelshkjony** [iBotPeaches/Apktool#3631] [Q&A] [answered]
I am working on a CLI project with apktool.jar Here I want to reduce the size of my executable. Please share specific proguard rules to apply for apktool.jar

Thanks for your awesome work 👌

**Comment by iBotPeaches** [comment, marked as answer]
I don't use Proguard anymore, but you can find some older rules here - https://github.com/iBotPeaches/Apktool/pull/3559/files
